ruby.md

```diff
@@ -756,26 +728,25 @@ Translations of the guide are available in the following languages:
    ```
 
 * <a name="method-invocation-parens"></a>
-  Use parentheses around the arguments of method invocations,
-  especially if the first argument begins with an open parenthesis `(`,
-  as in `f((3 + 2) + 1)`.
+  Use parentheses around the arguments of method invocations, when including
+  them improves clarity.
```

Comment:
Yeah, my _general_ guideline is to use parens with more than one argument, but that's because I tend to find that's when it helps improve clarity. I like leaving this discretionary, particularly since it doesn't change the shape of the code/squint test (unlike things involving indentation).